components/Modules/modules-main-content.tsx

```
@@ -149,10 +164,8 @@ export default function ModulesPageContent({
         const filtered = modules.filter(
           (module) =>
             module.name.toLowerCase().includes(searchQuery.toLowerCase()) ||
-            (module.description &&
-              module.description
-                .toLowerCase()
-                .includes(searchQuery.toLowerCase()))
+            (module.context &&
+              module.context.toLowerCase().includes(searchQuery.toLowerCase()))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)